README.md

```diff
@@ -1,4 +1,26 @@
-# Gender Pay Gap service
+# Gender Pay Gap Service
 
-This is the code for the [Gender Pay Gap service](https://gender-pay-gap.service.gov.uk)
+This is the code for the [Gender Pay Gap service](https://gender-pay-gap.service.gov.uk).
 
+From 2017, any organisation that has 250 or more employees must publish and report specific figures about their gender pay gap. 
+The gender pay gap is the difference between the average earnings of men and women, expressed relative to men’s earnings. 
+For example, ‘women earn 15% less than men per hour’.
+
+This service allows organisations to report their gender pay gap data. It then makes this data available to the general public. 
+It is managed by the Government Equalities Office (GEO) which forms part of the Cabinet Office.
+
+
+## Technical documentation
+
+This is a C# solution that has a user interface for public viewing pages, organisation reporting pages and an admin portal. There are also
+regular jobs for pulling information from Companies House and sending reminder emails. We use Identity Server for authentication 
+and SQL Server for the database.

```

Comment:
I struggled writing this and I'd be grateful for any feedback. How would you feel about providing the architecture diagram here?

Reply:
I think we should (at some point) create a documentation folder with zero-to-hero, architecture diagrams and details e.g. how to create a database migration.

This seems like a sensible description for now.